terraform/hcl/applicable/req_sw_terraform_aws_api_gateway_auth.tf

```diff
@@ -0,0 +1,6 @@
+resource "aws_api_gateway_method" "vulnerable_method" {
```

Comment:
[comment]: <> (FrogbotReview Iac 1 1 6 2 "vulnerable_method" /terraform/hcl/applicable/req_sw_terraform_aws_api_gateway_auth.tf)

## ![](https://raw.githubusercontent.com/jfrog/frogbot/master/resources/v2/applicableHighSeverity.png)<br>    high Infrastructure as Code Vulnerability 🛠️
	
Finding: authorization=NONE was detected

### 👇 Details

All `aws_api_gateway_method` resources should have `authorization != NONE`

Vulnerable example -
```
resource "aws_api_gateway_method" "vulnerable_example" {
  authorization = "NONE"
}
```

Secure example -
```
resource "aws_api_gateway_method" "secure_example" {
  authorization = "AWS_IAM"
}
```	

[[🐸 JFrog Frogbot]](https://github.com/jfrog/frogbot#readme)